conf/tidb.yml

```diff
@@ -216,3 +216,6 @@ binlog:
   # If IgnoreError is true, when writting binlog meets error, TiDB would stop writting binlog,
   # but still provide service.
   # ignore-error: false
+
```

Comment:
If tidb release 2.0 supports binlog socket, this config shoud be hide, see https://github.com/pingcap/tidb-ansible/blob/master/roles/tidb/templates/run_tidb_binary.sh.j2#L33.